HACKING.md

```diff
@@ -23,6 +23,7 @@ Install the necessary global dependencies.
 - postgresql
 - redis
 - phantomjs
+- libpq-dev
```

Comment:
what's this is for specifically? If it's ubuntu / linux specific we should add it to the platform-specific doc: https://github.com/legislated/legislated-api/blob/master/wiki/ubuntu.md.

Reply:
yea I think that's Ubuntu specific. I can move it over.